.github/CONTRIBUTING.md

```diff
@@ -127,3 +127,19 @@ You can replicate this process locally with the following commands:
 nf-core create -n testpipeline -d "This pipeline is for testing"
 nf-core lint nf-core-testpipeline
 ```
+
+## Codespaces
+
+This repo includes a devcontainer configuration which will create a GitHub Codespace for Nextflow development!
+
+Devcontainer specs:
+
+- [DevContainer config](.devcontainer/devcontainer.json)
+- [Dockerfile](.devcontainer/Dockerfile)
```

Comment:
```suggestion
> **Note:**
> Devcontainer specs:
>
> - [DevContainer config](.devcontainer/devcontainer.json)
> - [Dockerfile](.devcontainer/Dockerfile)
```

Reply:
Maybe worth moving down to the bottom of the section too, as most people won't care..

Reply:
Good idea.